app/app.go

```diff
@@ -727,17 +727,21 @@ func (app *App) GetSubspace(moduleName string) paramstypes.Subspace {
 	return subspace
 }
 
-func getNetworkInfo(app *App, ctx sdk.Context) (*NetworkConfig, error) {
+func getNetworkInfo(app *App, ctx sdk.Context, manifest *UpgradeManifest) (*NetworkConfig, error) {
 	// Load network config from file if given
 	var networkInfo *NetworkConfig
 	var err error
 	if app.cudosMigrationConfigPath != "" {
+		app.Logger().Info("Loading network config", "file", app.cudosMigrationConfigPath, "hash", app.cudosMigrationConfigSha256)
 		networkInfo, err = LoadNetworkConfigFromFile(app.cudosMigrationConfigPath, &app.cudosMigrationConfigSha256)
+		manifest.ConfigSource = fmt.Sprintf("file %s %s", app.cudosMigrationConfigPath, app.cudosMigrationConfigSha256)
```

Comment:
Manifest must  **\*not\***  conatin file path of input files - see [this comment](https://github.com/fetchai/fetchd/pull/383/files#r1786792275).